autoProcessTV/autoProcessTV.py

```diff
@@ -131,7 +134,10 @@ def processEpisode(dir_to_process, org_NZB_name=None):
         password_mgr = urllib2.HTTPPasswordMgrWithDefaultRealm()
         password_mgr.add_password(None, url, username, password)
         handler = HTTPBasicAuthHandler(password_mgr)
-        opener = urllib2.build_opener(handler)
+        if sys.version_info >= (2, 7, 9):
```

Comment:
If the version check can't be shared with the getURLFileLike can you at least make a common pythonVersionVerifiesHttpsCertificates or something similar please?